payments/swish/direct.md

```diff
@@ -0,0 +1,335 @@
+|         | └─➔&nbsp;`msisdn`             | `String`  | Number will be prefilled on payment page, if valid.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|             | └─➔&nbsp;`ecomOnlyEnabled`             | `boolean`  | `true` if to only enable Swish on e-commerce transactions.; otherwise `false` to also enable Swish transactions via mobile app.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+{:.code-header}
+**Response**
+
+```http
+POST /psp/swish/payments HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "payment": {
+        "id": "/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f",
+        "number": 992308,
+        "created": "2017-10-23T08:38:57.2248733Z",
+        "instrument": "Swish",
+        "operation": "Purchase",
+        "intent": "Sale",
+        "state": "Ready",
+        "currency": "SEK",
+        "amount": 0,
+        "description": "Test Purchase",
+        "payerReference": "AB1234",
+        "initiatingSystemUserAgent": "Mozilla/5.0",
+        "userAgent": "Mozilla/5.0...",
+        "language": "sv-SE",
+        "urls": {
+            "id": "/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f/urls"
+        },
+        "payeeInfo": {
+            "id": "/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f/payeeinfo"
+        }
+    },
+    "operations": [
+        {
+            "method": "PATCH",
+            "href": "http://localhost:18496/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f",
+            "rel": "update-payment-abort"
+        },
+        {
+            "method": "POST",
+            "href": "http://localhost:18496/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f/sales",
+            "rel": "create-sale"
+        }
+    ]
+}
+```
+
+## Sales
+
+The Sales resource lists the sales transactions (one or more) on a specific payment.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/swish/payments HTTP/1.1
```

Comment:
Why is there a second `POST` request against `/psp/swish/payments` here without an HTTP body?

Reply:
corrected this to be a GET request